helpers/validator/validator_test.go

```diff
@@ -0,0 +1,111 @@
+package validator
+
+import (
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+)
+
+type testPair struct {
+	value  string
+	result bool
+}
+
+var testsEmail = []testPair{
```

Comment:
Nit: I think _emailTests_, _messageTests_ and _nameTests_ might be easier to read.